src/components/partner/partner.service.ts

```diff
@@ -205,14 +204,14 @@ export class PartnerService {
     );
 
     const perms = await this.authorizationService.getPermissions({
-      resource: IProject,
+      resource: Partner,
       sessionOrUserId: session,
     });
 
     return {
       ...projectListOutput,
-      canRead: perms.member.canRead,
-      canCreate: perms.member.canEdit,
+      canRead: perms.projects.canRead,
```

Comment:
@CarsonF - This doesn't seem to be working as the read permissions here are coming back as false.  😶   Any ideas here?

Reply:
Need to grant access in the role files
https://github.com/SeedCompany/cord-api-v3/blob/e618a443ed797a7583bd87704842baead638de37/src/components/authorization/roles

Reply:
@CarsonF - I think I did this correctly.  Can you check it out?